test/Datadog.Trace.Tests/Configuration/ConfigurationSourceTests.cs

```diff
@@ -161,9 +161,27 @@ public void DefaultSetting(Func<TracerSettings, object> settingGetter, object ex
             // save original value so we can restore later
             var originalValue = Environment.GetEnvironmentVariable(key);
 
-            Environment.SetEnvironmentVariable(key, value, EnvironmentVariableTarget.Process);
-            IConfigurationSource source = new EnvironmentConfigurationSource();
-            var settings = new TracerSettings(source);
+            TracerSettings settings;
+
+            if (key == "DD_SERVICE_NAME")
```

Comment:
```suggestion
            if (key == "DD_SERVICE_NAME" || key == "DD_SERVICE")
```
We accept `DD_SERVICE` and `DD_SERVICE_NAME`. Can you also modify this branch to use `key` instead of `ConfigurationKeys.ServiceName`?

Reply:
Oh I see, you're trying to test that, even when `DD_SERVICE` is set in CI, that this `DD_SERVICE_NAME` test config takes effect. Nevermind, this should work as-is, ignore my comment